File: app/src/main/java/com/example/proyecto/models/Facturas.java
package com.example.proyecto.models;

import java.util.Date;

public class Facturas {
    private String descEstado;
    private Float importeOrdenacion;
    private Date fecha;

    public Facturas(String descEstado, Float importeOrdenacion, Date fecha) {
        this.descEstado = descEstado;
        this.importeOrdenacion = importeOrdenacion;
        this.fecha = fecha;
    }

    public String getDescEstado() {
        return descEstado;
    }

    public void setDescEstado(String descEstado) {
        this.descEstado = descEstado;
    }

    public Float getImporteOrdenacion() {
        return importeOrdenacion;
    }

    public void setImporteOrdenacion(Float importeOrdenacion) {
        this.importeOrdenacion = importeOrdenacion;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }
}

File: app/src/main/java/com/example/proyecto/models/Facturas.kt
package com.example.proyecto.models

import java.util.*
import kotlin.collections.ArrayList

data class Facturas(
    private final var  numFacturas:Double,
    private final var facturas:List<facturaindividual>
)
 data class facturaindividual(
    private final var descEstado:String,
    private final var importeOrdenacion:Long,
    private final var fecha:Date
)


